docs/podman-search.1.md

```diff
@@ -13,7 +13,9 @@ The user can specify which registry to search by prefixing the registry in the s
 **registires.search** table in the config file - **/etc/containers/registries.conf**.
 The number of results can be limited using the **--limit** flag. If more than one registry
 is being searched, the limit will be applied to each registry. The output can be filtered
-using the **--filter** flag.
+using the **--filter** flag. To get all available images in a registry without a specific
+query, the user can just enter the registry name with a trailing (example **registry.fedoraproject.org/**).
+Note, searching without a query will only work for registries that implement the v2 API.
```

Comment:
Down below you say the search only works for v2, but here you're saying it works for v1, but only the query works on v2?  I'm a bit confused.